script.js

```diff
@@ -1,3 +1,434 @@
+(function ($) {
```

Comment:
Please see if you can split the javascript up and put the editable-library in its own file.